packages/zoe/src/contractSupport/priceAuthorityTransform.js

```diff
@@ -26,18 +27,18 @@ export const mintQuote = async (
 ) => {
   const quoteAmount = {
     brand: quoteBrand,
-    value: [{ amountIn, amountOut, timer, timestamp }],
+    value: makeCopyBagFromElements([{ amountIn, amountOut, timer, timestamp }]),
   };
   const quotePayment = await E(quoteMint).mintPayment({
     brand: quoteBrand,
-    value: [quoteAmount],
+    value: makeCopyBagFromElements([quoteAmount]),
```

Comment:
The original makes no sense to me. I think my change is correctness-preserving, but it is also confusion preserving. The revised form make no sense to me either.